validator/client/beacon-api/beacon_api_validator_client.go

```diff
@@ -192,7 +193,7 @@ func (c *beaconApiValidatorClient) WaitForChainStart(ctx context.Context, _ *emp
 }
 
 func (c *beaconApiValidatorClient) StartEventStream(ctx context.Context, topics []string, eventsChannel chan<- *event.Event) {
-	eventStream, err := event.NewEventStream(ctx, c.jsonRestHandler.HttpClient(), c.jsonRestHandler.Host(), topics)
+	eventStream, err := event.NewEventStream(ctx, &http.Client{}, c.jsonRestHandler.Host(), topics) // event stream should not be subject to the same settings as other api calls
```

Comment:
This comment isn't clear, consider making a variable to make it more clear.
```suggestion
	client := &http.Client{} // event stream should not be subject to the same settings as other api calls, so we won't use c.jsonRestHandler.HttpClient()
	eventStream, err := event.NewEventStream(ctx, client, c.jsonRestHandler.Host(), topics) 
```